src/datasources/balances-api/coingecko-api.service.ts

```diff
@@ -92,12 +100,12 @@ export class CoingeckoApi implements IPricesApi {
         notFoundExpireTimeSeconds: this.defaultNotFoundExpirationTimeSeconds,
         expireTimeSeconds: this.pricesTtlSeconds,
       });
+      return result?.[nativeCoinId]?.[lowerCaseFiatCode];
```

Comment:
This is taken from the old `PricesRepository`, as this is Coingecko-specific, it's being pushed to `CoingeckoApi`.